### PR TITLE
#45 - dependencyManagement updated

### PR DESCRIPTION
**Pull request by olegmoz**
For #45 
Added `http` and `http-client` modules to `dependencyManagement`, `asto` version bumped

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -118,7 +118,17 @@ SOFTWARE.
       <dependency>
         <groupId>com.artipie</groupId>
         <artifactId>asto</artifactId>
-        <version>0.27</version>
+        <version>0.28</version>
+      </dependency>
+      <dependency>
+        <groupId>com.artipie</groupId>
+        <artifactId>http</artifactId>
```
Comment (olenagerasimova) [inline pom.xml:125]: @olegmoz hm, are you sure this will work? All our repositories (asto including) use ppom and http has asto as a dependency

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz one question, take a look, please

File: pom.xml (hunk 1 above)
Comment (olegmoz) [reply pom.xml:125]: @olenagerasimova `dependencyManagement` section just specifies the versions and does not add dependencies automatically. This works for `asto` now - the module does not get included into itself.

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks, for the explanation, good to merge

**Pull request merged**